src/ngraph/runtime/gpu/cudnn_emitter.cpp

```diff
@@ -1403,6 +1403,62 @@ size_t runtime::gpu::CUDNNEmitter::build_batchnorm(const cudnnBatchNormMode_t& b
     return this->m_primitive_emitter->register_primitive(batchnorm, hash);
 }
 
+size_t runtime::gpu::CUDNNEmitter::build_lrn(const cudnnLRNMode_t& lrn_op,
+                                             const std::string& dtype,
+                                             const Prop& direction,
+                                             const Shape& input_shape,
+                                             const Shape& output_shape,
+                                             const double lrn_alpha,
+                                             const double lrn_beta,
+                                             const double lrn_bias,
+                                             const size_t lrn_size)
+{
+    // construct hash to determine if kernel needs to be emitted
+    // or if it already exists in the primitive list
+    std::stringstream ss;
+    ss << "lrn_op_" << lrn_op << "_dtype_" << dtype << "_dir" << static_cast<int>(direction) << "_i"
+       << join(input_shape, "_") << "_o" << join(output_shape, "_") << "_alpha_" << lrn_alpha
```

Comment:
Is the output_shape necessary for the hash?

Reply:
No - it's in lockstep with input_shape. Reworked code to reflect this.